packages/bsky/src/services/moderation/status.ts

```diff
@@ -118,6 +127,11 @@ export const adjustModerationSubjectStatus = async (
     return null
   }
 
+  // Exit early if someone is trying to game the system by appealing someone else's content
+  if (isAppealEvent && createdBy !== subjectDid) {
+    return null
+  }
+
```

Comment:
This is the part i'm still least sure of myself, even after settling on this behavior after earlier discussion. It feels a little weird to special case like this (and re-write the createdBy elsewhere).

I think it's probably fine though, and changing the behavior probably won't be a change to Lexicons so we aren't particularly locked-in to these behaviors ("who can create appeal reports").

Reply:
Agree, I think we can iterate on it.  We might be able to push it up into authorization layer then forget about it here.